cocos/audio/audio-source.ts

```diff
@@ -345,13 +356,14 @@ export class AudioSource extends Component {
      */
     public play (): void {
         if (!this._isLoaded && this.clip) {
-            this._operationsBeforeLoading.push('play');
+            this._operationsBeforeLoading.push({ op: 'play', params: null });
```

Comment:
It is better not use magic string, such as `play`. It is better to define it as a const value, then other codes can just refer to the type. For example, may define it as 
```typescript
class AudioOperationType {
    const PLAY: string = 'play';
    ...
}
```